modules/light-clients/06-solomachine/types/update.go

```diff
@@ -93,16 +84,19 @@ func (cs ClientState) VerifyClientMessage(cdc codec.BinaryCodec, clientMsg expor
 	return nil
 }
 
-// update the consensus state to the new public key and an incremented sequence
-func update(clientState *ClientState, header *Header) (*ClientState, *ConsensusState) {
+// UpdateState updates the consensus state to the new public key and an incremented sequence.
+func (cs ClientState) UpdateState(
```

Comment:
We should probably add a basic test for this function. Or will that come later?

Reply:
I'll put a basic test cases in place and it can be added to once the storage ops are completed 👍